GOVERNANCE.md

```diff
@@ -0,0 +1,15 @@
+# Governance
+
+## Project Maintainers
+
+[Project maintainers](MAINTAINERS.md) are responsible for activities around maintaining and updating the papiNet API standard. Final decisions on the standard reside with the project maintainers.
+
+Maintainers MUST remain active. If they are unresponsive for >6 months, they will be automatically removed unless a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of the other project maintainers agrees to extend the period to be greater than 6 months.
+
+New maintainers can be added to the project by a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) vote of the existing maintainers. A potential maintainer may be nominated by an existing maintainer. A vote is conducted in private between the current maintainers over the course of a two weeks voting period. At the end of the two weeks, votes are counted and a pull request is made on the repository adding the new maintainer to the [MAINTAINERS.md](MAINTAINERS.md) file.
```

Comment:
I suggest the voting period on adding a particular new maintainer is extended to four weeks, since in at least Europe the Summer holiday is usually foour weeks or longer.

It is not that important that new maintainers are added swiftly.

Reply:
I am ambivalent on the voting period length,

Reply:
I will extend it to four weeks.